code/src/components/AlbumTitle.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const AlbumTitle = (props) => {
+  return (
+    <span className="album-name">
```

Comment:
Is there a reason for using a span element to contain the link? If you remove it, you will still only have one jsx-element in the function. We checked the CSS-file and could not find this className being used.